### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sarbull**
Signed-off-by: Sirbu Nicolae-Cezar <sirbunicolaecezar@gmail.com>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,12 +34,14 @@
...
-    "@lfai/egeria-ui-components": "^4.0.7",
-    "@lfai/egeria-ui-core": "^4.0.3",
-    "@mantine/core": "4.2.9",
-    "@mantine/hooks": "4.2.9",
-    "@mantine/next": "4.2.9",
+    "@emotion/cache": "^11.10.5",
+    "@emotion/react": "^11.10.5",
+    "@emotion/serialize": "^1.1.1",
+    "@lfai/egeria-js-commons": "^4.0.10",
+    "@lfai/egeria-ui-components": "^4.0.13",
```
[the remark this answers is not in the document]
Comment (sarbull) [reply package.json:41]: @sonatype-lift ignore

File: Dockerfile [hunk 2]
```diff
@@ -1,32 +1,38 @@
...
 
-FROM public.ecr.aws/nginx/nginx:latest
+WORKDIR /app
+COPY . /app
+RUN npm install
+RUN npm run build:docker
 
-ARG version=1.4.0
+# Stage 2 - Containerize
+FROM nginx:1.19.0
```
[the remark this answers is not in the document]
Comment (planetf1) [reply Dockerfile:10]: we should also check (after building) to see what the image security looks like (quay.io will do this for us)

File: Dockerfile [hunk 3]
```diff
@@ -1,32 +1,38 @@
-# SPDX-License-Identifier: Apache-2.0
-# Copyright Contributors to the Egeria project
+# Stage 1 - Build
+FROM node:16.13.0 AS builder
 
-FROM public.ecr.aws/nginx/nginx:latest
+WORKDIR /app
+COPY . /app
+RUN npm install
+RUN npm run build:docker
```
Comment (planetf1) [inline Dockerfile:7]: How does this change the docker build? any impact on supporting multiple architectures?

**Review by planetf1 — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
[the remark this answers is not in the document]
Comment (sarbull) [reply Dockerfile:13]: ok

File: .github/workflows/node-build.yml [hunk 4]
```diff
@@ -65,7 +65,6 @@ jobs:
           tags: odpi/egeria-ui:${{ steps.package-version.outputs.current-version}}, quay.io/odpi/egeria-ui:${{ steps.package-version.outputs.current-version}}
           context: .
           file: ./Dockerfile
-          platforms: linux/amd64,linux/arm64
```
[the remark this answers is not in the document]
Comment (sarbull) [reply .github/workflows/node-build.yml:68]: i got the build failing for some reason on these platforms

[the remark this answers is not in the document]
Comment (sarbull) [reply .github/workflows/node-build.yml:68]: NPM usually works with both

[the remark this answers is not in the document]
Comment (planetf1) [reply .github/workflows/node-build.yml:68]: we need to have both, so if there are issues we need to figure out.
Some additional setup is needed for cross-building. the ci/cd scripts already have this setup (but you need to reinstate the lines above)

**Comment by sarbull** [comment]
Moving docker changes in a different PR

File: .github/workflows/node-build.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (sarbull) [reply .github/workflows/node-build.yml]: moved this to https://github.com/odpi/egeria-ui/pull/468

**Pull request merged**